### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mheadd**
Added some additional details to the technical requirements section.

### CODE REVIEW

**Comment by mheadd** [comment]
I should have indicated in this PR that I opened a [separate issue](https://github.com/AlaskaDHSS/acq-alaska-dhss-modernization/issues/222) for the changes to the `DevOps.md` file.

**Comment by ctro** [comment]
@mheadd I pushed a commit to this PR that just links to the updated DevOps document.

File: README.md [hunk 1]
```diff
@@ -143,15 +143,9 @@ Key documents
 * Link to the Worker Portal User Acceptance Testing Environment: https://uat.myaries.alaska.gov 
 
 ## Technology requirements
-We are working in the Microsoft universe. We prefer:
+We are working with the Microsoft technology stack. Please see [DevSecOps.md](/DevSecOps.md) for specific requirements.
```
Comment (ctro) [inline README.md]: There is another PR open which renames that document to DevSecOps: https://github.com/AlaskaDHSS/acq-alaska-dhss-modernization/pull/227

Once that PR merges then this link will be correct.

Comment (mheadd) [reply README.md]: Cool - thanks @ctro!

Comment (ctro) [reply README.md]: OTOH, if it makes more sense to ditch the DevSecOps document and just paste that contents under "Technology requirements" let me know..

Comment (mheadd) [reply README.md]: Nah, I think the DevSecOps doc makes sense.

**Comment by mheadd** [comment]
#227 has been merged to master so this should be good to go once @waldoj approves.

**Pull request merged**